### DISCUSSION TITLE
GitHub Actions + Vercel for GitHub (only the comment bot)

### DISCUSSION THREAD

**Discussion by irangarcia** [vercel/community#1137] [Help]
### Summary

Hey!

There is some way that I can create preview deployments through the [Github Actions](https://vercel.com/guides/how-can-i-use-github-actions-with-vercel#configuring-github-actions-for-vercel) and at the same time have the Vercel bot only for commenting preview URLs?

I took a look at some previous discussions but I just found a GitHub Action [plugin](https://github.com/orgs/vercel/discussions/761
) that is not maintained by Vercel.

**Comment by swarnava** [comment]
GitHub action based deployment is something we don't support officially. You might want to ask [here](https://github.com/amondnet/vercel-action/discussions) to find more details!

**Reply by irangarcia** [reply]
But the tutorial on [GitHub Actions for Vercel](https://vercel.com/guides/how-can-i-use-github-actions-with-vercel#configuring-github-actions-for-vercel)  is not the same as `vercel-action`, right?

I didn't know about `vercel-action`, I was talking about the workflow described on your tutorial

**Reply by swarnava** [reply]
Correct it's not same. It's a workaround to support custom workflow and we can only help issue to debug deployment specific issue.

For advance usecase like commenting automatically (which is responsibility of GH action), you might want to consider checking with GitHub support.
